### PR TITLE
interruption handling stories

### PR DESCRIPTION
**Pull request by melindaloubser1**
Added stories with different kinds of interruptions to improve exception handling

### CODE REVIEW

File: data/nlu.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - yeah
 - yes please
 - yes plz
+- /affirm
```
Comment (ArjaanBuijk) [inline data/nlu.md:11]: Is the `/` intentionally included?

File: data/nlu.md [hunk 2]
```diff
@@ -200,3 +202,5 @@
 - I want to pay Bill
 - pay my friend
 - transfer money
+- I want to transfer $100 to John
```
[the remark this answers is not in the document]
Comment (melindaloubser1) [reply data/nlu.md:205]: no, these are picked up by spacy, so they're not annotated =)

File: data/nlu.md [hunk 3]
```diff
@@ -200,3 +202,5 @@
...
+- i want to transfer $400 to Akela
```
[the remark this answers is not in the document]
Comment (melindaloubser1) [reply data/nlu.md:206]: no, these are picked up by spacy, so they're not annotated =)

File: data/nlu.md (hunk 1 above)
Comment (melindaloubser1) [reply data/nlu.md:11]: nope this was a mistake

Comment (melindaloubser1) [reply data/nlu.md:11]:
```suggestion
```

File: data/nlu.md [hunk 4]
```diff
@@ -29,6 +30,7 @@
 - what is my account balance
 - what's my account balance?
 - what's my account balance
+- What's my account [balance](payment_amount)?
```
[the remark this answers is not in the document]
Comment (melindaloubser1) [reply data/nlu.md:33]: nope mistake sorry!
```suggestion
- What's my account balance?
```

**Pull request merged**